toolchain/check/import.cpp

```diff
@@ -421,4 +440,147 @@ auto ImportLibrariesFromOtherPackage(Context& context,
   }
 }
 
+// Looks up a name in an imported scope. An `identifier` is provided if
+// `name_id` corresponds to an identifier in the current file; otherwise,
+// `name_id` is file-agnostic and can be used directly.
```

Comment:
```suggestion
// Looks up a name in a scope imported from another package. An `identifier` is provided if
// `name_id` corresponds to an identifier in the current file; otherwise,
// `name_id` is file-agnostic and can be used directly.
```
I think this is only handling the cross-package case (due to the access check).